docs/_jan-milestone_.md

```diff
@@ -0,0 +1,57 @@
+# Jan milestone
+
+## Overview
+
+- Take some time to:
+  - fix bugs, improve developer experience, fix docs
+  - triage open issues across all the repos, not just `loopback-next`
+- Migration work continues to be one of the focus
+- Focus on issues/PRs for loopback-connector-mongodb
+
+## Goals
+
+- [ ] Migration guide between LB3 and LB4 MVP #453
+
+  - [ ] [3] How to migrate datasources from LB3 to LB4 #3946
+  - [ ] [3] How to migrate boot scripts #3957
+
+- [ ] Improve developer experience
+
+  - [ ] [3] Expose isActive() method in Transaction interface #3471
+  - [ ] [3] Missing model.ts after running lb4 openapi command #4135
+  - [ ] [2] Model strict: false is ignored with Postgre SQL repository #4042
+
+- [ ] Bugs
+
+  - [ ] [3] Docs: Improve TodoList Tutorial instructions and completed tutorial
+        'lb4 example todo-list' #4161
+  - [ ] [Needs Estimates] lb3 - Relations on uuid
+        https://github.com/strongloop/loopback/issues/4287
+
+- [ ] [3][spike] API Connect / LoopBack 4 integration #4115
+
+- [ ] [Needs estimates] chore: update dependency change-case to v4 #4233
+
+## Stretch goals
+
+- [ ] Spike: migrate tests to Jest (ts-jest) #3159
```

Comment:
Let's remove this item. Based on my discussion with @boneskull, we would like to try few experiments with Mocha to see if we can get DX that's good enough.